### PR TITLE
feat: add tests for python example application

### PR DESCRIPTION
**Pull request by rkpattnaik780**
Some methods are temporarily defined as the example application have not been moved to the `app-services-guide` repository.

### CODE REVIEW

**Comment by henryZrncik** [comment]
look good to me but branch needs to rebase and even ifi go with that rebase it fails with following response

```
Traceback (most recent call last):
  File "/private/var/folders/_n/4v4gm05x5652b269h5fz_yfh0000gp/T/quickstart-repo10683828265679183106/rhosak_example_codes/python_example/consumer.py", line 4, in <module>
    from confluent_kafka import Consumer
ModuleNotFoundError: No module named 'confluent_kafka'

Step failed
java.io.IOException: Not able to find the produced message
	at io.managed.services.test.quickstarts.steps.PythonSteps.python_example_consumer_should_print_the_produced_message(PythonSteps.java:136)
	at ✽.Python example consumer should print the produced message(file:///Users/hzrncik/Documents/code/work/e2e-test-suite/src/test/resources/io/managed/services/test/quickstarts/quickstarts.feature:57)

```

File: src/test/resources/io/managed/services/test/quickstarts/quickstarts.feature [hunk 1]
```diff
@@ -25,6 +25,37 @@ Feature: Quick starts for OpenShift Streams for Apache Kafka
...
+    Then the Kafka topic prices is listed in the topics table
+
+    # 4. Running the Python example producer
+    Given the Kafka instance is in Ready state
+    Given you have set the permissions for your service account to produce and consume from topic prices
+    Then you run Python example producer
+
+    # 4. Running the Python example consumer
+    Given the Kafka instance is in Ready state
+    Then Python example consumer should print the produced message
```
Comment (henryZrncik) [inline src/test/resources/io/managed/services/test/quickstarts/quickstarts.feature:57]: failed step.

**Review by henryZrncik — CHANGES_REQUESTED** [review]

File: src/test/resources/io/managed/services/test/quickstarts/quickstarts.feature (hunk 1 above)
Comment (rkpattnaik780) [reply src/test/resources/io/managed/services/test/quickstarts/quickstarts.feature]: Have split the step like in rest of snippets test.

**Review by henryZrncik — CHANGES_REQUESTED** [review]

File: src/test/java/io/managed/services/test/quickstarts/steps/PythonSteps.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        while ((output = stdInput.readLine()) != null) {
+            stdOutput += output;
+            System.out.println(output);
+        }
+    }
+
+    @Then("Python example consumer should print proper message")
+    public void pythonExampleConsumerShouldPrintProperMessage() throws Exception {
+        if (!stdOutput.contains("Shore")) {
+            throw new IOException("Not able to find the produced message");
```
[the remark this answers is not in the document]
Comment (rkpattnaik780) [reply src/test/java/io/managed/services/test/quickstarts/steps/PythonSteps.java:140]: I guess we need to have this dependencies installed in the jenkins environment. Is there any existing implementation I can refer to.

**Review by henryZrncik — APPROVED** [review]
runs  correctly.

**Pull request merged**